app/src/main/java/com/ahgitdevelopment/course/customexamples/features/screens/datastore/timer/TimerViewModel.kt

```diff
@@ -69,7 +64,6 @@ class TimerViewModel @Inject constructor(
         object : CountDownTimer(this, INTERVAL) {
             override fun onTick(millisUntilFinished: Long) {
                 _elapsedTime.value = millisUntilFinished
-//                saveTimer(millisUntilFinished)
                 Log.i("Timer", "onTick millisUntilFinished $millisUntilFinished")
```

Comment:
Remove unnecessary logs added just for tracing the app and do not add much value.